benchmarking/commons/hpo_main_local.py

```diff
@@ -158,11 +158,14 @@ def create_objects_for_tuner(
         benchmark=benchmark,
         extra_args=None if extra_args is None else extra_metadata(args, extra_args),
     )
+    tuner_name = args.experiment_tag
+    if args.use_long_tuner_name_prefix:
+        tuner_name += f"-{args.benchmark}-{seed}"
```

Comment:
Rather than passing this additional metadata inside `tuner_name`, why not create an explicit separate variable, something like `tuner_output_path`?

Reply:
`tuner_name` is an argument of `Tuner`, it is the prefix of the result files written by the tuner.